bibliopixel/remote/server.py

```diff
@@ -6,8 +6,10 @@
 import logging
 from .. import log
 
-werk_log = logging.getLogger('werkzeug')
-werk_log.setLevel(logging.ERROR)
+# I want to keep this to supress the "Running on" message
```

Comment:
Fair enough. I actually have to keep it in... otherwise you don't get any of the HTTP server messages like what files it's serving or that /api/run_animation/ was called. IDK... maybe users don't care.
How about this... I disable the werk log unless --verbose is used on run?